chain-tx-enclave-next/mls/src/ciphersuite.rs

```diff
@@ -278,7 +274,7 @@ impl CipherSuite {
 
     /// TODO: use generic array?
     pub fn encrypt_group_info(
-        &self,
+        self,
```

Comment:
just &[u8] is fastest, not necessary to use generic array

Reply:
generic array is meant rather for welcome_key, welcome_nonce ... not the (variable-length) plaintext input